src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -517,22 +517,26 @@ function showVersionWarningBanner(data) {
   inner.appendChild(button);
   const skipLink = document.getElementById("pst-skip-link");
   skipLink.after(outer);
-  const autoHeight = outer.offsetHeight;
-
+  // At least 3rem height
+  const autoHeight = Math.min(
```

Comment:
this should be `Math.max`, otherwise if `outer.offsetHeight` is less than 3rem it will still be set to that smaller value

```suggestion
  const autoHeight = Math.max(
```